example.js

```diff
@@ -46,11 +46,16 @@ console.log('\n\n' + boxen(chalk.black('unicorn'), {
 }) + '\n');
 
 const sentences = 'Unbreakable_text_because_it_has_no_spaces '.repeat(5);
-console.log('\n\n' + boxen(sentences, {align: 'left'}) + '\n');
+console.log('\n\n' + boxen(sentences, {textAlignement: 'left'}) + '\n');
```

Comment:
Typo
```suggestion
console.log('\n\n' + boxen(sentences, {textAlignment: 'left'}) + '\n');
```

Reply:
Oh, that one is a bit shameful. Sorry for that, as you can tell I am french and I didn't know that alignment was spelled this way.